src/main/index.js

```diff
@@ -74,13 +77,15 @@ const makeFullUrl = (url, search = null) => {
  * Prompt in a platform-specific way for permission to access the microphone or camera, if Electron supports doing so.
  * Any application-level checks, such as whether or not a particular frame or document should be allowed to ask,
  * should be done before calling this function.
+ * This function may return a Promise!
  *
  * @param {string} mediaType - one of Electron's media types, like 'microphone' or 'camera'
- * @returns {boolean} - true if permission granted, false otherwise.
+ * @returns {boolean|Promise.<boolean>} - true if permission granted, false otherwise.
```

Comment:
Why not just always return a promise?

Reply:
We talked about this separately but just to record it here: functionally this is fine because it's called with `await`, which hides the difference between returning a promise and returning a raw Boolean. To avoid possible future problems I will make a followup change where this function always returns a promise, especially since there's basically no down-side to doing so in this context. (This isn't performance-critical, no synchronous work is happening around this call, etc.)